opentaskspal/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,9 @@
+<manifest xmlns:tools="http://schemas.android.com/tools"
+        package="org.dmfs.opentaskspal">
+
+    <!--TODO Remove after https://github.com/dmfs/opentasks/issues/392-->
+    <uses-sdk tools:overrideLibrary="org.dmfs.android.contentpal"/>
```

Comment:
please add a note to #392 about this.

Reply:
added: https://github.com/dmfs/opentasks/issues/392#issuecomment-329198715